osu.Game/Rulesets/Mods/IApplicableToAudio.cs

```diff
@@ -1,6 +1,5 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
```

Comment:
please label commits better. this should be something like "Add missing licence header"